questionnaire/questionnaire.js

```diff
@@ -2114,7 +2140,13 @@ module.exports = {
                 type: 'final'
             },
             'p-applicant-you-cannot-get-compensation': {
-                type: 'final'
+                on: {
+                    ANSWER: [
+                        {
+                            target: 'p-applicant-did-the-crime-happen-once-or-over-time'
```

Comment:
Is this no longer a final section?

Reply:
No, I thought it was, but apparently the applicant can continue from here. The thinking is that we cannot prevent the applicant from applying just because they've not reported it. There's a very strong chance they'll be rejected in this case, hence the warning, but I believe there are circumstances they would still be eligible. The routing for this has been amended so that going down this path navigates the user through the app without asking any police questions.

Reply:
Business rules dictate this change. Happy to go with it.